data/i18n/zh-cn/zh-cn.toml

```
@@ -28,11 +35,38 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+[cve_id]
+other = "CVE ID"
+
+[cve_issue_url]
+other = "CVE GitHub Issue URL"
+
+[cve_json_external_url]
+other = "external_url"
+
+[cve_json_id]
+other = "id"
+
+[cve_json_summary]
+other = "概要"
+
+[cve_json_url]
+other = "url"
+
+[cve_summary]
+other = "问题描述"
+
+[cve_table]
+other = "Kubernetes CVE 列表"
+
+[cve_url]
+other = "CVE URL"
```

Seems this term is not used at this time.

```suggestion
other = "CVE 網址"
```

---

We can revisit this one when the upstream English version is changed.